CMGTools/TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -9,98 +9,117 @@
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : _susy2lss_multiIso_withMiniIsoRelaxed_ConePtJetPtRatio(lep),
-                lambda lep : (lep.mvaIdPhys14 > 0.73+(0.57-0.73)*(abs(lep.eta)>0.8)+(+0.05-0.57)*(abs(lep.eta)>1.479) or abs(lep.pdgId)!=11),
-                lambda lep : lep.sip3d<4,
-                lambda lep : abs(lep.dxy)<0.05,
-            ]) ) )
-MODULES.append ( ('leptonFakeRateFO4', lambda : ObjTagger('FO4','LepGood',
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : _susy2lss_multiIso_withMiniIsoRelaxed_CutForFO4(lep),
-                lambda lep : (lep.mvaIdPhys14 > 0.73+(0.57-0.73)*(abs(lep.eta)>0.8)+(+0.05-0.57)*(abs(lep.eta)>1.479) or abs(lep.pdgId)!=11),
-                lambda lep : lep.sip3d<4,
-                lambda lep : abs(lep.dxy)<0.05,
-            ]) ) )
-MODULES.append ( ('leptonFakeRateFO1InSitu', lambda : ObjTagger('FO1InSitu','LepGood',
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : (lep.mvaIdPhys14 > 0.73+(0.57-0.73)*(abs(lep.eta)>0.8)+(+0.05-0.57)*(abs(lep.eta)>1.479) or abs(lep.pdgId)!=11),
-                lambda lep : lep.sip3d>=4,
-            ]) ) )
-MODULES.append ( ('leptonFakeRateFO2InSitu', lambda : ObjTagger('FO2InSitu','LepGood',
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : lep.sip3d>=4,
-            ]) ) )
-MODULES.append ( ('leptonFakeRateFO3InSitu', lambda : ObjTagger('FO3InSitu','LepGood',
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : _susy2lss_multiIso_withMiniIsoRelaxed_ConePtJetPtRatio(lep),
-                lambda lep : (lep.mvaIdPhys14 > 0.73+(0.57-0.73)*(abs(lep.eta)>0.8)+(+0.05-0.57)*(abs(lep.eta)>1.479) or abs(lep.pdgId)!=11),
-                lambda lep : lep.sip3d>=4,
-            ]) ) )
-MODULES.append ( ('leptonFakeRateFO4InSitu', lambda : ObjTagger('FO4InSitu','LepGood',
-            list_cuts_tightlepid_nomultiiso_noeltightmvaid_nosip_nodxy+[
-                lambda lep : _susy2lss_multiIso_withMiniIsoRelaxed_CutForFO4(lep),
-                lambda lep : (lep.mvaIdPhys14 > 0.73+(0.57-0.73)*(abs(lep.eta)>0.8)+(+0.05-0.57)*(abs(lep.eta)>1.479) or abs(lep.pdgId)!=11),
-                lambda lep : lep.sip3d>=4,
-            ]) ) )
-
-#from CMGTools.TTHAnalysis.tools.vertexWeightFriend import VertexWeightFriend
-#pufile="../path/to/nvtx/file.root"
-#MODULES.append ( ('puWeights', VertexWeightFriend(pufile,pufile,"nvtx_signal","nvtx_data",verbose=True) ) )
+MODULES.append ( ('leptonFakeRateFO2', ObjTagger('FO2','LepGood',[
+                lambda lep: lep.miniRelIso<0.4,
+                lambda lep: _susy2lss_lepId_CBloose(lep),
+                lambda lep: _susy2lss_lepId_loosestFO(lep),
+                lambda lep: _susy2lss_lepId_IPcuts(lep),
+                ] ) ) )

```

Comment:
it probably doesn't matter since `ObjTagger` doesn't have any heavy initialization function, but why did you remove the lambda wrapping it?